### ISSUE TITLE
Has issue from dotnet/corert issue 8241 and dotnet/runtimelab pull 98 resurfaced?

### ISSUE THREAD

**Issue by klcantrell** [dotnet/runtimelab#497]
The original post from [dotnet/corert#8241](https://github.com/dotnet/corert/issues/8241) mentioned a problem loading `System.Security.Cryptography.Native.Apple`. Reading through the thread, I was under the impression the problem was fixed by [dotnet/runtimelab#98](https://github.com/dotnet/runtimelab/pull/98), but I am still encountering the following error when running my example app:

```
Unhandled Exception: System.AggregateException: One or more errors occurred. (The SSL connection could not be established, see inner exception.)
 ---> System.Net.Http.HttpRequestException: The SSL connection could not be established, see inner exception.
 ---> System.Security.Authentication.AuthenticationException: Authentication failed, see inner exception.
 ---> System.DllNotFoundException: Unable to load DLL 'libSystem.Security.Cryptography.Native.Apple.dylib': The specified module could not be found.
```

I'm not sure how much of the below is helpful, but here is info on my setup:

**.net: 5.0.101**
**macOS: 10.15.7**
**Microsoft.DotNet.ILCompiler: 6.0.0-alpha.1.20622.2**
#### .csproj
```xml
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>net5.0</TargetFramework>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="Microsoft.DotNet.ILCompiler" Version="6.0.0-*" />
  </ItemGroup>

</Project>
```
#### code
```csharp
using System;
using System.Net.Http;
using System.Threading.Tasks;

namespace Test
{
    class Program
    {
        private static readonly HttpClient client = new HttpClient();

        static void Main(string[] args)
        {
            SendRequest().Wait();
        }

        private static async Task SendRequest()
        {
            var response = await client.GetStringAsync("https://jsonplaceholder.typicode.com/todos/1");
            Console.WriteLine(response);
            return;
        }
    }
}
```

**Comment by jkotas** [comment]
Thank you for reporting this. Yes, there appears to be a regression introduced by #454

**Comment by jkotas** [comment]
Fixed by #499. We have an issue with publishing official builds, so it may take some time until the fix shows up on the nuget feed.

**Comment by klcantrell** [comment]
I’m amazed by the quick response. Thank you @jkotas! 💯
